roles/jenkins/tasks/main.yml

```diff
@@ -70,5 +71,33 @@
     state: restarted
   become: yes
   when: installed_plugins.changed
+  register: restart_status
+- import_tasks: wait_for_jenkins.yml
```

Comment:
you can do 
```
- import_tasks: wait_for_jenkins.yml
  when: installed_jenkins.changed
```
and put the restart of jenkins inside the import as well

Reply:
done, also renamed wait_for_jenkins.yml to restart_jenkins.yml